src/content/faq/General.md

```diff
@@ -8,20 +8,76 @@ questions: Why should I use GraphQL?,Is GraphQL a database language like SQL?,Do
+
 ## Is GraphQL frontend or backend?
 
+Neither. Instead, it’s [how you can exchange information between the two](https://www.howtographql.com/basics/3-big-picture/). Both the client and the server should understand and communicate through the GraphQL specification.
+
 ## Does GraphQL replace Redux or other state management libraries?
 
+No, GraphQL isn’t a state management library - but it can reduce the need for one.
+
+One benefit of state management libraries like Redux is that they can manipulate API responses into a format that your application understands. With GraphQL, you have control over [what data you request](/learn/queries/#fields) and [how that data is formatted](/learn/execution/). So this benefit is already built-in. Many [client libraries](https://graphql.org/code/#graphql-clients) can also be used to manage state and have features like caching built-in. You may still decide to implement a state management library, but using it to format response data won’t be necessary.
+
 ## Is GraphQL only for React or JavaScript developers?
 
+No, not at all. [GraphQL services can be written in any language](/learn/schema/#type-language). Our [Code page](/code/) contains a long list of libraries in many different programming languages to help with that.
+
+It’s understandable why you’d think this, though. GraphQL was introduced at a [React conference](https://www.youtube.com/watch?v=9sc8Pyc51uU) and [GraphQL.js](/graphql-js/) is one of the most widely used reference implementations to date. We know this can be confusing, so we’re working to improve our documentation and add more code samples that aren’t written in JavaScript.
+
 ## What is a GraphQL client and why would I use one?
 
+GraphQL clients can help you handle [queries, mutations,](/learn/queries/) and [subscriptions](https://spec.graphql.org/draft/#sec-Subscription) to a [GraphQL server](https://www.howtographql.com/advanced/1-server/). This could be done manually, but it would require a lot of work depending on the amount of data you have. By utilizing the underlying structure of a GraphQL API, clients can abstract away some of these tedious processes. Features that can be automated include batching, caching, UI updates, build-time schema validation, and many more. 
+
+You can find a list of GraphQL clients in various languages on our [Code page](/code/#graphql-clients). There’s also an [in-depth explanation of their benefits](https://www.howtographql.com/advanced/0-clients/) on How To GraphQL.
+
+It takes more time to implement a GraphQL client upfront, but it becomes worth it the more features you need. You might want to start out [using HTTP requests](/learn/serving-over-http/) as the underlying transport layer (here’s [an example in JavaScript](/graphql-js/graphql-clients/)), then switch to a more intricate client service as your application grows in complexity.
```

Comment:
I am not sure what "more time to implement a GraphQL client upfront" means.

Reply:
I meant like... GraphQL clients come with many features out-of-the-box and it can be difficult to figure out what you need when you're first working with them. I'd argue that implementing a GraphQL client (Apollo for instance) is more complicated than doing HTTP requests initially. But I can see why it'd be confusing in this context. 

Do you think it'd make sense to cut this sentence and start with `You might want to start...`? This could address both this comment and the previous.

Reply:
I think cutting the sentence makes sense.

Separately, "using HTTP requests as the underlying transport layer" doesn't feel clear enough to me (since most of the GraphQL clients also use HTTP requests as the underlying transport layer), can we somehow make this more obvious that we mean issuing HTTP requests manually? Maybe something like: "You might want to start out by issuing GraphQL results with a regular HTTP client, later switching to a GraphQL optimised client as your application grows in complexity."

(Sorry for my UK English :wink:)

Reply:
So much better - thank you @benjie!